openpype/hosts/flame/hooks/pre_flame_setup.py

```diff
@@ -99,8 +101,39 @@ def execute(self):
 
         app_arguments = self._get_launch_arguments(data_to_script)
 
+        # fix project data permission issue
+        self._fix_permissions(project_name, volume_name)
+
         self.launch_context.launch_args.extend(app_arguments)
 
+    def _fix_permissions(self, project_name, volume_name):
+        """Work around for project data permissions
+
+        Reported issue: when project is created locally on one machine,
+        it is impossible to migrate it to other machine. Autodesk Flame
+        is crating some unmanagable files which needs to be opened to 0o777.
+
+        Args:
+            project_name (str): project name
+            volume_name (str): studio volume
+        """
+        dirs_to_modify = [
+            "/usr/discreet/project/{}".format(project_name),
+            "/opt/Autodesk/clip/{}/{}.prj".format(volume_name, project_name),
+            "/usr/discreet/clip/{}/{}.prj".format(volume_name, project_name)
+        ]
+
+        for dirtm in dirs_to_modify:
+            for root, dirs, files in os.walk(dirtm):
+                try:
+                    for d in dirs:
+                        os.chmod(os.path.join(root, d), self.permisisons)
+                    for f in files:
+                        os.chmod(os.path.join(root, f), self.permisisons)
+                except OSError as _E:
+                    self.log.warning("Not able to open files: {}".format(_E))
```

Comment:
It's a little bit longer but I would skip all dirs or files which already have expected permissions.
```suggestion
                try:
                    for name in set(dirs) | set(files):
                        path = os.path.join(root, name)
                        st = os.stat(path)
                        if oct(st.st_mode) != self.permissions:
                            os.chmod(path, self.permisisons)

                except OSError as exc:
                    self.log.warning("Not able to open files: {}".format(exc))
```